site/_resources/upgrading.md

```diff
@@ -9,21 +9,66 @@ toc: h1,h2
-Contour 1.1.0 is the current stable release.
+Contour 1.2.0 is the current stable release.
 
 <div class="alert-deprecation">
-<h3><b>Deprecation Notice</b></h3>
-<p>The <code>IngressRoute</code> CRD has been deprecated and will not receive further updates. Contour 1.1.0 continues to support the IngressRoute API, however we anticipate it will be removed in the future.</p>
-<p>Please see the documentation for <a href="{% link docs/v1.1.0/httpproxy.md %}"><code>HTTPProxy</code></a>, which is the successor to <code>IngressRoute</code>.
-You can also read the <a href="{% link _guides/ingressroute-to-httpproxy.md %}">IngressRoute to HTTPProxy upgrade</a> guide.</p>
+<h3>Deprecation Notice</h3>
+<p>The <code>IngressRoute</code> CRD has been deprecated and will not receive further updates. Contour 1.2.0 continues to support the IngressRoute API, however we anticipate it will be removed in the future.</p>
+<p>Please see the documentation for <a href="{% link docs/{{site.latest}}/httpproxy.md %}"><code>HTTPProxy</code></a>, which is the successor to <code>IngressRoute</code>. You can also read the <a href="{% link _guides/ingressroute-to-httpproxy.md %}">IngressRoute to HTTPProxy upgrade</a> guide.</p>
 </div>
 
 &nbsp;
 
 ## Recommended Envoy version
 
+All users should ensure the Envoy image version is `docker.io/envoyproxy/envoy:v1.13.0`.
+
+Please see the [Envoy Release Notes][17] for information about issues fixed in Envoy 1.13.0.
+
+## The easy way to upgrade
+
+If the following are true for you:
+
+ * Your installation is in the `projectcontour` namespace.
+ * You are using one of the [example][1] deployments.
+ * Your cluster can take few minutes of downtime.
+
+Then the simplest way to upgrade to 1.2.0 is to delete the `projectcontour` namespace and reapply one of the example configurations.
+From the root directory of the repository:
+
+```
+$ kubectl delete namespace projectcontour
+$ kubectl apply -f examples/<your-desired-deployment>
+```
+
+This will remove both the Envoy and Contour pods from your cluster and recreate them with the updated configuration.
+If you're using a `LoadBalancer` Service, (which most of the examples do) deleting and recreating may change the public IP assigned by your cloud provider.
+You'll need to re-check where your DNS names are pointing as well, using [Get your hostname or IP address][12].
+
+**Note:** If you deployed Contour into a different namespace than `projectcontour` with a standard example, please delete that namespace.
+Then in your editor of choice do a search and replace for `projectcontour` and replace it with your preferred name space and apply the updated manifest.
+
+## The less easy way
+
+This section contains information for administrators who wish to apply the Contour 1.1.0 to 1.2.0 changes manually.
+
+### Upgrade to Contour 1.2.0
+
+Change the Contour image version to `docker.io/projectcontour/contour:v1.2.0`.
+
+TODO(stevesloka): add information for configuring the Envoy shutdown manager.
```

Comment:
ShutdownManager has docs which explain it's configuration. The default examples and rendered examples have the manager configured in the Envoy DS. Might be able to just link to those docs and call out the change?